script/Ciel/Day 5/05_01_CIEL05_2_2.txt

```diff
@@ -23,7 +23,7 @@ Any normal living being would be mortally wounded just by approaching him.
 [sha:a2e0c20d3dd6f43b31b79c1a64041dc784e49884]{
 -- Page 301, Offset 11670.
 -- それを俺はホテルで一度味わっている……！
-I already got a taste of that once back at the hotel...!
+I already got a taste of that once, back at the hotel...!
```

Comment:
I'd either not add the comma, or do `...once before, back...`. Straight once to comma just feels slightly off.

Reply:
this is another linebreak-driven change

Reply:
I'm leaving it as is then